core-build/gulp-core-build/tsconfig.json

```diff
@@ -17,6 +17,7 @@
       "es5",
       "scripthost",
       "es2015.promise",
+      "es2015.iterable",
```

Comment:
>"es2015.iterable", [](start = 6, length = 18)

This package does not appear to need it.

Reply:
I made sure that nothing had it, but then I added it back only if the build broke without having it... I think something with the `"target": "es6"` vs `"lib": "es5"` causes it.